lib/themes/dosomething/paraneue_dosomething/scss/content/_reportback.scss

```diff
@@ -157,4 +160,23 @@
     }
   }
 
-}
\ No newline at end of file
+}
+
+
+
+// @TODO: move to proper locations
+// Temporary additions to work through fix for upload button issues.
+input[type="file"] {
```

Comment:
I can place in `.reportback` for now, but we should eventually move into Neue to be universal for file inputs.